test/fixtures/simple-app/mixins/time-stamps.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Timestamping",
+  "version": "0.0.1",
```

Comment:
IMO the version should stay in `package.json`, but since this metadata is not used by boot nor loopback, then it does not really matter.

However, in order to make future maintenance less expensive and also to make the tests easier to understand, the test data should include only things that are absolutely necessary for the test to run. 

Please trim down this file down to the minimal version needed by the tests.